test-framework/jacoco/runtime/src/main/java/io/quarkus/jacoco/runtime/JacocoConfig.java

```diff
@@ -11,10 +11,11 @@
 public class JacocoConfig {
 
     /**
-     * The jacoco data file
+     * The jacoco data file. By default this will be target/jacoco-quarkus.exec.
```

Comment:
Could we use `defaultValueForDocumentation` instead?

Reply:
I didn't remember that at all. @vsevel fancy doing a new PR with this improvement?

Reply:
Sure. I will add some info on the guide as well